isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,355 @@
+    printf "anaconda versions for a label at https://anaconda.org/usgs-astrogeology/isis"
+    printf ", examples include 8.0.3 (LTS/Feature), 8.2.0_RC1 (RC), and 2025.02.22 (dev)"
+    exit 1
+}
+
+POSITIONAL_ARGS=()
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -h|--help)
+      print_help
+      ;;
+    -n|--env_name)
+      ENV_NAME="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -m|--miniforge_dir)
+      MINIFORGE_DIR="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -p|--data_prefix)
+      ISISDATA_PREFIX="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    --no_data)
+      DOWNLOAD_DATA=NO
+      shift # past argument
+      ;;
+    -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift # past argument
+      ;;
+  esac
+done
+
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+# If a ISIS LABEL was given, set it
+if [ -n "$1" ]; then
+    ISIS_LABEL=$1
+else
+    printf "Defaulting to $ISIS_LABEL"
+fi
+
+if [ "$ISIS_LABEL" = "main" ]; then
+    PACKAGE_NAME="usgs-astrogeology::isis"
+else
+    PACKAGE_NAME="usgs-astrogeology/label/$ISIS_LABEL::isis"
+fi
+
+# If a ISIS VERSION was given, set it
+if [ -n "$2" ]; then
+    ISIS_VERSION=$2
+fi
+
+printf "\nBeginning install of $PACKAGE_NAME\n\n"
+
+# Determine the OS type
+case "$(uname)" in
+    "Linux")
+        MINIFORGE_INSTALLER="Miniforge3-Linux-x86_64.sh"
+        LIBGL_INSTALL=""
+        ;;
+    "Darwin")
+        MINIFORGE_INSTALLER="Miniforge3-MacOSX-x86_64.sh"
+        LIBGL_INSTALL=""
+        export CONDA_SUBDIR=osx-64
+        ;;
+    *)
+        echo "Unsupported OS: $(uname)"
+        exit 1
+        ;;
+esac
+
+# Install Miniforge if it's not already installed
+if ! command -v conda &> /dev/null
+then
+    echo "Miniforge not found, installing Miniforge..."
+
+    # If a MINIFORGE_DIR is not set, ask the user
+    if [ -z "$MINIFORGE_DIR" ]; then
+        MINIFORGE_DIR="$HOME/miniforge3"
+
+        printf "Miniforge will be installed at this location:\n\n" 
+        printf "\t$MINIFORGE_DIR/\n\n"
+        printf "\\n"
+        printf "  - Press ENTER to confirm the Miniforge install location\\n"
+        printf "  - Press CTRL-C to abort the installation\\n"
+        printf "  - Or specify a different location below\\n"
+        printf "\\n"
+        printf "[%s] >>> " "$MINIFORGE_DIR"
+
+        read -r miniforge_install_path
+
+        # If input was given, set it
+        if [ -n "$miniforge_install_path" ]; then
+            MINIFORGE_DIR=$miniforge_install_path
+        fi
+    fi
+
+    curl -kL "https://github.com/conda-forge/miniforge/releases/latest/download/$MINIFORGE_INSTALLER" -o Miniforge3.sh || failed_command "Miniforge download"
+    bash Miniforge3.sh -b -p $MINIFORGE_DIR || failed_command "Miniforge installation"
+    export PATH="$MINIFORGE_DIR/bin:$PATH"
+    mamba init || exit 1
```

Comment:
Why `mamba` and not `conda` here?

Reply:
To some degree they are interchangeable

Reply:
Fair enough. The script seems to use both `mamba` and `conda`. Consider picking one and using consistently throughout even though they are interchangeable in this application.